varya/assets/css/variables.css

```diff
@@ -35,7 +35,7 @@
 	/* Line Height */
 	--global--line-height-base: 1;
 	--global--line-height-body: 1.7;
-	--global--line-height-heading: 1.125;
+	--global--line-height-heading: 1.3;
```

Comment:
I adjusted the value of this global variable to match the H1 - H3 line height from the comps.

Reply:
Coool. Makes sense 👍